docs/Using-Operations-Service.md

```diff
@@ -210,6 +212,49 @@ func reject(operation: WMTOperation, reason: WMTRejectionReason) {
 }
 ```
 
+## Detail of an Operation
+
+To get a detail of an operation based on operation ID use `WMTOperations.getDetail`. Operation detail is confirmed by the possession factor so there is no need for creating  `PowerAuthAuthentication` object. The returned result is the operation and its current status.
+
+```swift
+import WultraMobileTokenSDK
+import PowerAuth2
+
+// Reject operation with some reason
+func getDetail(operationId: String, reason: WMTRejectionReason) {
+    operationService.getDetail(operationId: operationId) { error in
+        switch result {
+        case .success(let operation):
+            // process operation
+            break
+        case .failure(let error):
+            // process error
+            break
+        }
+    }
+}
+```
+
+## Claim an Operation
+
```

Comment:
Let's add a block of text here that explains what a non-personalized operation is (+ example).